hardhat.config.ts

```diff
@@ -151,15 +156,18 @@ const config: HardhatUserConfig = {
             {
                 version: "0.4.12",
             },
+            {
+                version: "0.8.2",
```

Comment:
What's the reason for adding this? Is there some file that is locked to version 0.8.2 for its the compiler? All of our files use 0.8.11, ad it looks like there wasn't any `*.sol` file touched in this PR, so this seems unnecessary if we were able to compile before this PR